moj-node/server/routes/index.js

```diff
@@ -13,17 +14,20 @@ module.exports = function Index({
       logger.info('GET index');
 
       const { establishmentId } = req.app.locals.envVars;
+      const { bookingId } = (req.session && req.session.user) || 0;
```

Comment:
use Ramda's `path` ?
Also why is does bookingID default to 0?

Reply:
Done!